source/gameengine/VideoTexture/Texture.cpp

```diff
@@ -479,10 +479,19 @@ int Texture::pyattr_set_mipmap(void *self_v, const KX_PYATTRIBUTE_DEF *attrdef,
 		PyErr_SetString(PyExc_TypeError, "The value must be a bool");
 		return -1;
 	}
+	if (!self->m_source) {
```

Comment:
Hi, we have to call videoffmpeg constructor to before texture.Texture(...).mipmap = True because we have to know if the image type is video to forbid to set mipmapping on videos. If you do this:
tex = texture.Texture(own, 0, 0)
tex.mipmap = True
tex.source = texture.VideoFFmpeg(...)

this will lead to a crash. This is the reason why I added this check: if (!self->m_source)